src/pkg/cli/client/byoc/clouds/aws.go

```diff
@@ -209,8 +209,19 @@ func (b *ByocAws) Deploy(ctx context.Context, req *defangv1.DeployRequest) (*def
 	}
 	b.cdTasks[etag] = taskArn
 
+	deployed, err := b.GetServices(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve deployed services: %w", err)
+	}
+	for _, si := range deployed.Services {
+		domain := si.Service.Domainname
+		if si.UseAcmeCert {
```

Comment:
This flag is set by us in BYOC, so why do we need `GetServices`?

Reply:
Agreed, not needed as mentioned above.